pkg/storegateway/series_chunks_test.go

```diff
@@ -628,7 +619,7 @@ func TestLoadingSeriesChunksSetIterator(t *testing.T) {
 				s.release()
 			}
 
-			assert.Zero(t, int(bytesPool.balance.Load()))
+			assert.Zero(t, chunkBytesSlicePool.(*pool.TrackedPool).Balance.Load())
```

Comment:
Can we also assert `chunkBytesSlicePool.(*pool.TrackedPool).Gets.Load()` is greater than 0, to ensure something was requested from the pool?

Reply:
Fixed